install.sh

```diff
@@ -313,23 +319,16 @@ fi
 echo -e "${BLUE}
 To set the default PAGER environment variable to use less:
 echo \"export PAGER='less'\" >> ~/.bashrc && . ~/.bashrc
-
```

Comment:
Any reason to remove spaces? I think they are better to read with spaces:)

Reply:
Just ran this through shellcheck, which found an actual legitimate bug, (probably an error in the last commit of mine), so I took their other recommendations as well.

Reply:
Oh! thought youu were talking about some other space. (Sorry android github app is lacking here.)

I'm agnostic as to this space. I'll add it back in.

Reply:
Run `git diff` before committing? ;)

Reply:
I was actually editing in geany, and then pasting into the web interface for uploading to the specific branch. Normally I check the diff there. I must have hit a geany command to remove spaces somehow. 

The obvious explanation is PEBKAP, but in fairness that could also be connected to Bluetooth issues with my keyboard. I'm getting a hardwired keyboard today, so maybe that will help.

Reply:
(The Pixel Slate keyboard has finally almost died due to a crappy flat ribbon cable to the USB connector shearing as is apparently common. Google's long term hardware support... not great...)